Packs/ManageEngine_PAM360/Integrations/ManageEnginePAM360/ManageEnginePAM360_description.md

```diff
@@ -0,0 +1,4 @@
+## ManageEngine PAM360
+PAM360 is a comprehensive solution for businesses looking to incorporate PAM into their overall security operations. PAM360's robust core functionalities and contextual integration capabilities enables IT admins take a holistic approach to privileged access security via a central console facilitating a deep correlation of privileged access data and overall network data, resulting in meaningful inferences and quicker remedies.
+
+In order to use this product, you will need to add a server URL. The default port used by PAM360 is 8282.
```

Comment:
Please add information about all parameters needed in order to configure the integration (APP-token for instance) and how to get them.

Reply:
Done